star-rx72n-firmware/lib/rx_hal/src/rspi.c

```
@@ -149,31 +148,31 @@ rx_err_t rspi_init_peripheral(uint8_t channel, uint8_t mode, bool use_16bit)
   system_regs()->prcr = k_rx_prcr_unlock_prc1_prc3;
 
   if (channel == k_rspi_channel_0) {
```

Clang (14.0.6)

[warning] repeated branch in conditional chain

(bugprone-branch-clone)